internal/config/config.go

```diff
@@ -13,32 +13,29 @@ import (
 // NodeConfig contains all the Node variables.
 // See the corresponding environment variable for the variable documentation.
 type NodeConfig struct {
-	LogLevel                               LogLevel
-	LogPrettyEnabled                       bool
-	BlockchainID                           uint64
-	BlockchainHttpEndpoint                 Redacted[string]
-	BlockchainWsEndpoint                   Redacted[string]
-	LegacyBlockchainEnabled                bool
-	EvmReaderDefaultBlock                  DefaultBlock
-	EvmReaderRetryPolicyMaxRetries         uint64
-	EvmReaderRetryPolicyMaxDelay           Duration
-	BlockchainBlockTimeout                 int
-	ContractsInputBoxAddress               string
-	ContractsInputBoxDeploymentBlockNumber int64
-	SnapshotDir                            string
-	PostgresEndpoint                       Redacted[string]
-	HttpAddress                            string
-	HttpPort                               int
-	FeatureClaimSubmissionEnabled          bool
-	FeatureMachineHashCheckEnabled         bool
-	Auth                                   Auth
-	AdvancerPollingInterval                Duration
-	ValidatorPollingInterval               Duration
-	ClaimerPollingInterval                 Duration
-	EspressoBaseUrl                        string
-	EspressoStartingBlock                  uint64
-	EspressoNamespace                      uint64
-	EspressoServiceEndpoint                string
+	LogLevel                       LogLevel
+	LogPrettyEnabled               bool
+	BlockchainHttpEndpoint         Redacted[string]
+	BlockchainWsEndpoint           Redacted[string]
+	LegacyBlockchainEnabled        bool
+	EvmReaderDefaultBlock          DefaultBlock
+	EvmReaderRetryPolicyMaxRetries uint64
+	EvmReaderRetryPolicyMaxDelay   Duration
+	BlockchainBlockTimeout         int
+	SnapshotDir                    string
+	PostgresEndpoint               Redacted[string]
```

Comment:
BlockchainID - node config - ok
ContractsInputBoxAddress - node config - ok
ContractsInputBoxDeploymentBlockNumber - node config - ok
SnapshotDir - node config - ok

We will probably end up cleaning more configs in the future...